gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #234: fix interested/going override bug

### Description

I forgot to write code that joins two squads' interested/going events. The logic is as follows:

* If an event is marked as going for either squad, the resulting squad will have that event marked as going.
* If an event is marked as interested for either squad (not going in either), the resulting squad will have that event marked as interested.

### Testing Directions

```
./setup
python manage.py runserver
```

Please see `showup/tests.py` for the edge cases I wrote. I wrote comments there so it should be easy to follow along and test for yourself on the UI. Note that we have an invariant which states that the squad with the smaller squad id remains.

### Checklist
